pallets/settlement/src/lib.rs

```diff
@@ -1962,13 +1973,15 @@ impl<T: Config> Module<T> {
     fn ensure_venue_filtering(
         tickers: &mut BTreeSet<AssetID>,
         asset_id: AssetID,
-        venue_id: &VenueId,
+        venue_id: &Option<VenueId>,
```

Comment:
Seems a bit odd to have it optional here, rather not calling this function when it is None?